vmms/ec2SSH.py

```diff
@@ -102,6 +110,13 @@ def instanceName(self, id, name):
         """
         return "%s-%d-%s" % (config.Config.PREFIX, id, name)
 
+    def keyPairName(self, id, name):
```

Comment:
Any reason this function is `instanceName()`. They seem to have a similar function